webapp/app/forms/steps/lotse/merkzeichen.py

```diff
@@ -124,21 +120,16 @@ class InputForm(SteuerlotseBaseForm):
             validators=[ValidDisabilityDegree()],
             render_kw={'help': _l('form.lotse.field_person_beh_grad-help'),
                        'data_label': _l('form.lotse.merkzeichen.disability_degree')})
-        person_b_has_merkzeichen_g = BooleanField(
-            render_kw={'data_label': _l('form.lotse.merkzeichen.has_merkzeichen_g.data_label')},
-            name='person_b_has_merkzeichen_g')
-        person_b_has_merkzeichen_ag = BooleanField(
-            render_kw={'data_label': _l('form.lotse.merkzeichen.has_merkzeichen_ag.data_label')},
-            name='person_b_has_merkzeichen_ag')
-        person_b_has_merkzeichen_bl = BooleanField(
-            render_kw={'data_label': _l('form.lotse.merkzeichen.has_merkzeichen_bl.data_label')},
-            name='person_b_has_merkzeichen_bl')
-        person_b_has_merkzeichen_tbl = BooleanField(
-            render_kw={'data_label': _l('form.lotse.merkzeichen.has_merkzeichen_tbl.data_label')},
-            name='person_b_has_merkzeichen_tbl')
-        person_b_has_merkzeichen_h = BooleanField(
-            render_kw={'data_label': _l('form.lotse.merkzeichen.has_merkzeichen_h.data_label')},
-            name='person_b_has_merkzeichen_h')
+        person_b_has_merkzeichen_g = MerkzeichenBooleanField(
+            render_kw={'data_label': _l('form.loMerkzeichenBooleanFieldtse.merkzeichen.has_merkzeichen_g.data_label')})
```

Comment:
I think that was a typo:
```suggestion
            render_kw={'data_label': _l('form.lotse.merkzeichen.has_merkzeichen_g.data_label')})
```